File: count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int sum = 0;
        for(String word: words){
            if(isConsistent(word,allowed)){
                sum++;
            }
        }
        return sum;
    }

    public boolean isConsistent(String word, String allowed){
        for(char c: word.toCharArray()){
            if(!allowed.contains(Character.toString(c))){
                return false;
            }
        }
        return true;
    }
}

File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        count = 0
        for word in words:
            didFail = False
            for ch in word:
                if ch not in allowed:
                    didFail = True
                    break
                    
            if not didFail:
                count += 1

        return count